data/mods/Aftershock/README.md

```diff
@@ -1,10 +1,11 @@
+* As it becomes a total conversion mod it will look very different from the base game of Cataclysm: Dark Days Ahead and may not interact kindly with other mods.  If you've been away for a while you may wonder where some parts of Aftershock have gone.  In the process of becoming a total conversion mod we are eliminating items that no longer fit the theme.  For example, all the Mad Max aesthetics are currently removed.  For them to come back would require someone to write up a location/faction that made them make sense in a futuristic extrasolar colony.  
+Aftershock is being developed into a full conversion mod set in a mostly abandoned extrasolar colony named Salus 4.  The PC has arrived on Salus for various reasons depending on their profession.  Eventually we hope to have multiple potential end game situations such as finding what you came here for and leaving the planet with enough money to retire to more secret endings.  There will no longer be zombies but instead a variety of alien and human based enemies.  Toe to toe combat will likely be more deadly in Aftershock with more emphasis on picking your battles and only fighting when absolutely necessary.  Special locations may have a different combat aesthetic for example you may enter a dungeon like zone with hordes of small weak enemies that can't be avoided.  As part of this concept where different situations require different solutions we have implemented bionic slots but made the process of changing out CBMs much easier with less chance of failure.  This is to encourage players to change their loadout based on where they are going and what they are doing.
 
 ### Areas seeking contributors
-Faction interactions and new factions.  Currently there are two new factions in Aftershock. PrepNet and Whately Clan.  PrepNet is at war with the Mycus and the Whately's are mad scientists.  
-Uplifted animal mobs and mutation lines are always wanted.  I'd love to see someone make a Rabbit or Jackelope line.  
-Hi-tech item recipes.  Aftershock has it's own special crafting for ultratech and we'd love to see what you come up with.
+Faction interactions and new factions.  Currently there are two new factions in Aftershock. PrepNet and the Whately Clan.  PrepNet is seeking to build an independent colony and the Whatelys are mad scientists.  
+Uplifted animal mobs and mutation lines are always wanted.  I'd love to see someone make a Rabbit or Jackelope line.  Currently we have Mi-Go, Mastodon, and Cecaelian mutation trees.
+Hi-tech item recipes.  Aftershock has its own special crafting for ultratech and we'd love to see what you come up with.
```

Comment:
We can figure this out later, but we do want this to state the limit on the hi-tech items, by having a clear difference between tier 4 human tech/scrap (stuff that can be built today) tier 5 scap/tech (stuff that was prior to the fall, and required super advanced AI to manufacture) and alien tech, which I imagine will not be craft-able.